.circleci/config.yml

```diff
@@ -492,7 +492,23 @@ jobs:
 
   node-jest: *node-plugin-base
 
-  node-knex: *node-plugin-base
+  node-knex:
+    <<: *node-plugin-base
+    steps:
+      - run:
+          name: Install older python for sqlite, when needed
```

Comment:
Is this needed even with [better-sqlite3](https://www.npmjs.com/package/better-sqlite3)?

Reply:
Yes, because older versions of `knex` do not support `better-sqlite3`.